cg/constants/pacbio.py

```diff
@@ -1,24 +1,30 @@
 """Constants related to PacBio sequencing."""
 
+from cg.constants import FileExtensions
+
 
 class PacBioDirsAndFiles:
-    ADAPTER_REPORT: str = "adapter.report.json"
-    BASECALLING_REPORT: str = "ccs.report.json"
+    CCS_REPORT_SUFFIX: str = "ccs_report.json"
```

Comment:
Why is this the only one with _SUFFIX?
```suggestion
    CCS_REPORT: str = "ccs_report.json"
```

Reply:
That file is called `m84202_240522_135641_s1.ccs_report.json`, so in this case it is a suffix